lib/epi_deploy/git_wrapper.rb

```diff
@@ -60,6 +60,10 @@ def tag_list(options = {limit: 5})
       @tag_list ||= `git for-each-ref --sort=taggerdate --format '%(tag)' refs/tags`.gsub("'", '').split.reverse
     end
 
+    def current_branch
+      git.current_branch
```

Comment:
Is it worth caching the current branch here?

Reply:
I'm a little worried that epi_deploy might change branch while still using the same object. If it does do that then caching could cause unexpected errors. I'm not certain if this is a situation that will ever happen though.

Reply:
Ah OK, makes sense. The whole process isn't particularly transactional, which could cause issues if the branch changes unexpectedly, but I don't think it's too much of an issue